src/api/app/models/event/request.rb

```diff
@@ -70,11 +70,14 @@ def calculate_diff(a)
     end
   end
 
-  def payload_with_diff
-    source_projects = payload['actions'].map { |action| action[:source_project] }.uniq.compact
-    source_projects.each do |source_project|
-      return payload if Project.unscoped.is_remote_project?(source_project, true)
+  def source_from_remote?
```

Comment:
this will return `true` or `nil`, I would prefer returning `true` or `false`. So:

``` Ruby
def source_from_remote?
  payload['actions'].each do |action|
    return true if Project.unscoped.is_remote_project?(action['sourceproject'], true)
  end
  false
end
```

Reply:
done